test/backend/db_tests.js

```diff
@@ -0,0 +1,747 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// require('ass')
```

Comment:
Why is "ass" no longer required?  Is it because we don't run this ourselves, but just export it for the backends to run?